src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleEllipsoidTissueInputValidation.cs

```diff
@@ -47,34 +43,35 @@ private static ValidationResult ValidateGeometry(IList<LayerTissueRegion> layers
             // check that layer definition is valid
             var tempResult = MultiLayerTissueInputValidation.ValidateLayers(layers);
 
-            if (!tempResult.IsValid){ return tempResult; }
+            if (!tempResult.IsValid) return tempResult;
 
-            if ((ellipsoid.Dx == 0) || (ellipsoid.Dy == 0) || (ellipsoid.Dz == 0))
+            if (ellipsoid.Dx == 0 || ellipsoid.Dy == 0 || ellipsoid.Dz == 0)
             {
                 tempResult = new ValidationResult(
                     false,
                     "SingleEllipsoidTissueInput: ellipsoid has a radial axis equal to 0",
                     "SingleEllipsoidTissueInput: make sure Dx, Dy, Dz are > 0");
             }
 
-            if (!tempResult.IsValid) { return tempResult; }
+            if (!tempResult.IsValid) return tempResult;
 
             // test for air layers and eliminate from list
             var tissueLayers = layers.Where(layer => !layer.IsAir());
 
             // check that there is at least one layer of tissue 
-            if (!tissueLayers.Any())
+            var layerTissueRegions = tissueLayers.ToList();
+            if (!layerTissueRegions.Any())
             {
                 tempResult = new ValidationResult(
                     false,
                     "SingleEllipsoidTissueInput: tissue layer is assumed to be at least a single layer with air layer above and below",
                     "SingleEllipsoidTissueInput: redefine tissue definition to contain at least a single layer of tissue");
             }
 
-            if (!tempResult.IsValid) { return tempResult; }
+            if (!tempResult.IsValid) return tempResult;
 
             // check that ellipsoid contained within a tissue layer
-            bool correctlyContainedInLayer = tissueLayers.Any(
+            var correctlyContainedInLayer = layerTissueRegions.Any(
                 layer =>

```

Comment:
Same question as above - did this code functionality change?

Reply:
This one is okay as is.